opcua/common/manage_nodes.py

```diff
@@ -85,7 +85,13 @@ def create_variable(parent, *args):
     """
```

Comment:
@destogl instantiate is implemented using low level calls since we need
complete flexibility there to copy all attribute. But it is not finished....

On Wed, Jul 6, 2016, 10:02 Denis Štogl <notifications@github.com> wrote:

> In opcua/common/manage_nodes.py
> <https://github.com/FreeOpcUa/python-opcua/pull/235#discussion_r69688758>:
>
> > @@ -85,7 +85,13 @@ def create_variable(parent, *args):
> >      """
>
> @oroulet <https://github.com/oroulet>
>
> I think we should move that code to me method create _ variable _ type
>
> We need this here for instantiate_object function anyway
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/FreeOpcUa/python-opcua/pull/235/files/20a394e891394054faff3f2ed1b3d7548f59a13c#r69688758>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACcfzm9gsqRAWV-R2Ww1Z6WVG_coAYxQks5qS2D_gaJpZM4JFcMH>
> .
>